packages/shared/src/hooks/analytics/useAnalyticsQueue.ts

```diff
@@ -16,7 +16,10 @@ export type PushToQueueFunc = (events: AnalyticsEvent[]) => void;
 
 const ANALYTICS_ENDPOINT = `${apiUrl}/e`;
 
-export default function useAnalyticsQueue(): {
+type UseAnalyticsQueueProps = {
+  method: typeof fetch;
```

Comment:
`method` is not explicit enough. Maybe `fetchLike` or `customFetch`, or anything else that hint this is a fetch method

Reply:
How does `fetchMethod` sound like?
I don't want to make it sound like it's only the proxy as it can also be the "regular" fetch

Reply:
Makes sense ☺️